curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-functional-programming-by-building-a-spreadsheet/6434759f78ec812264ff8f34.md

```diff
@@ -7,7 +7,7 @@ dashedName: step-16
 
 # --description--
 
-In your callback, call `createLabel()` and pass `number` as the argument. You should see some numbers appear in your spreadsheet.
+In your callback, make two function calls. Call `createLabel()` and pass `number` as the argument. You should see some numbers appear in your spreadsheet.
 
 Then call the `.forEach()` method on your `letters` array. Pass an empty callback function which takes a `letter` parameter.
```

Comment:
Here, we should specific that the answer needs to be nested inside the callback

```suggestion
Below your `createLabel` function call, call the `.forEach()` method on your `letters` array. Pass an empty callback function which takes a `letter` parameter.
```

Reply:
The "two function calls" part above should imply that forEach is nested.

Reply:
Again, I'm okay with your suggestion. But the  code you say meets the specification doesn't have two function calls in the callback, it has one within and one without. 
If you'd still want me to do it your way, sure. I don't mind.

Reply:
leave can leave the original as is